content/docs/faq.md

```diff
@@ -11,13 +11,20 @@ aliases: [/faq]
 
 Thanks for your interest in Packit Service! In order to start using the
 service, your repository or namespace needs to be allowed.
-For each installation we create a new issue in our
-[allowlist tracker](https://github.com/packit/notifications/issues/)
-to resolve this.
 Just be aware that we are now onboarding Fedora contributors who have
 a valid [Fedora Account System](https://fedoraproject.org/wiki/Account_System) account.
-So provide us enough information (your FAS account) in the issue.
-Until that, you will get a neutral status with `Namespace is not allowed!` message on your commits.
+
+We require our new users to have `GitHub Username` field set in the FAS account so that we can automatically
+check for the match between the `GitHub Username` field in the  
+FAS account and the Github account that triggers the verification. 
+For each installation, we create a new issue in our
```

Comment:
Does this happen in every case, or only when the GitHub and FAS usernames are different?

Reply:
Good point, I changed it a little bit, PTAL

Reply:
Looks better, thanks!